test_rest_endpoint.py

```diff
@@ -229,7 +229,7 @@ def test_find_url_method(self):
             'RequestMapping' : element_map_mock
         }
         url_expected, method_expected = self.rest_url_process.find_url_method(operation_info_mock)
-        url_actual = 'MockPathVariableName'
+        url_actual = '/rest/MockPathVariableName'
```

Comment:
( I know you haven't introduced this but ) isn't actual and expected swapped in these variable names? I am not sure in what order the assertEquals contract expects them, but the "_expected" var is assigned the Actual Thing.

Reply:
Yes, it is wrong all over the tests - I have noticed it and decided to postpone the correction, since there will be a lot of changes.